cdp_scrapers/prime_gov_utils.py

```diff
@@ -1,14 +1,20 @@
+import re
```

Comment:
can you do my nitpicky thing of adding the:

```
#!/usr/bin/env python
# -*- coding: utf-8 -*-
```
 comment at the top so that other machines that aren't setup in some standard fashion don't freak out haha.